### DISCUSSION TITLE
its crashing my saves

### DISCUSSION THREAD

**Discussion by jimmyG1** [maruohon/litematica#769] [General]
[crash-2023-10-05_15.16.17-client.txt](https://github.com/maruohon/litematica/files/12822976/crash-2023-10-05_15.16.17-client.txt)

**Comment by maruohon** [comment]
Seems like some of the initialization code probably didn't run normally when loading into the world. It's probably a mod conflict with something else, but no idea with what. But the first thing I'd try is without Optifine.

**Reply by jimmyG1** [reply]
that was not it

**Reply by jimmyG1** [reply]
so have you figured out what is causing it as Optifine is not the problem

**Reply by maruohon** [reply]
No, I'm not going to download a big pile of mods to try to reproduce every crash issue people have. That would be a massive time sink. If you want to use Litematica, then *you* have to try to find out which other mod is causing issues with it.

You can do that in a reasonable amount of time by "bisecting", so remove (= move away from the mods folder) half of the mods at a time (and use a separate test world so you don't break your regular world when removing mods), and when you know which half of the mods still causes the crash, then keep halving that set of mods until you find the hopefully one mod that is the issue. Doing this you should be able to narrow it down to one mod in less than about 15 game starts even with a pack of 250 mods. Ofc one thing that is going to be annoying in bisecting is keeping the required dependencies for all mods...

**Reply by jimmyG1** [reply]
its VanillaFix thank you can look into it to find out y
